### PR TITLE
ci: simplify code with unparam linter

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: mixer/adapter/stackdriver/metric/bufferedClient_test.go [hunk 1]
```diff
@@ -194,6 +194,7 @@ func createRetryPushFn(pushCount *int, expReqTS [][]*monitoring.TimeSeries, with
 				len(expReqTS), len(withError), len(failedTS), *pushCount)
 			return nil
 		}
+		// nolint: unparam
 		defer func(pc *int) {
 			(*pushCount)++
```
Comment (kyessenov) [inline mixer/adapter/stackdriver/metric/bufferedClient_test.go]: @bianpengyuan : is this a real bug with not using `pc` variable?

Comment (kyessenov) [reply mixer/adapter/stackdriver/metric/bufferedClient_test.go]: Added a fix. Thanks.

**Comment by rshriram** [comment]
did you just manually fix all 66 files?

**Comment by kyessenov** [comment]
contributes to #9933

**Comment by kyessenov** [comment]
@rshriram can you lgtm? We want to stop further linter breakages, so it would be good to enable this check.

**Pull request merged**